src/img/icons/caret-right-small.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="6" height="10" viewBox="0 0 6 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think, usually `viewBox` should be set to `0 0 30 30` to have consistency throughout the app (you would need to change the whole svg though)